File: src/main/java/com/mrpowergamerbr/loritta/userdata/MusicConfig.java
package com.mrpowergamerbr.loritta.userdata;

import lombok.Getter;
import lombok.Setter;

import java.util.ArrayList;
import java.util.List;

@Getter
@Setter
public class MusicConfig {
	public boolean isEnabled = false;
	public String musicGuildId = null;
	public boolean hasMaxSecondRestriction = true;
	public int maxSeconds = 420;
	public boolean autoPlayWhenEmpty = false;
	public List<String> urls = new ArrayList<String>();
	public boolean voteToSkip = true;
	public int required = 75;
}

File: src/main/java/com/mrpowergamerbr/loritta/userdata/MusicConfig.kt
package com.mrpowergamerbr.loritta.userdata

class MusicConfig {
	var isEnabled: Boolean = false
	var musicGuildId: String? = null
	var hasMaxSecondRestriction: Boolean = true
	var maxSeconds: Int = 420
	var autoPlayWhenEmpty: Boolean = false
	var urls: MutableList<String> = mutableListOf<String>()
	var voteToSkip: Boolean = true
	var required: Int = 75
	var allowPlaylists: Boolean = false
	var logToChannel: Boolean = false
	var channelId: String? = null
}